packages/gitmoji-changelog-core/src/index.spec.js

```diff
@@ -218,6 +218,98 @@ describe('changelog', () => {
     // but output should only has 3, since the 3rd is empty
     expect(changes).toHaveLength(3)
   })
+
+  describe('Handle prerelease versions like alpha, beta, rc...', () => {
+    describe('For init mode', () => {
+      it('should merge commits of previous prerelease if there is a matching release', async () => {
+        mockGroup([sparklesCommit]) // v1.0.0-alpha
+        mockGroup([recycleCommit]) // v1.0.0-beta
+        mockGroup([lipstickCommit]) // v1.0.0
+        mockGroup([]) // next version
+
+        gitSemverTags.mockImplementation(cb => cb(null, ['v1.0.0-alpha', 'v1.0.0-beta', 'v1.0.0']))
+
+        const { changes } = await generateChangelog({ mode: 'init', mergePrerelease: true })
+
+        // the project has 3 tags (2 prerelease and 1 release)
+        // output should only has 1, since 3rd one is the final release of the 2 other prereleases
+        expect(changes).toHaveLength(1)
+        expect(changes[0].version).toBe('1.0.0')
+        // TODO check merged commits count
+      })
+
+      it('should not merge commits of previous prerelease if no matching release found', async () => {
+        mockGroup([sparklesCommit]) // v1.0.0
+        mockGroup([recycleCommit]) // v1.1.0-alpha
+        mockGroup([lipstickCommit]) // v1.1.0-beta
+        mockGroup([]) // next version
+
+        gitSemverTags.mockImplementation(cb => cb(null, ['v1.0.0', 'v1.1.0-alpha', 'v1.1.0-beta']))
+
+        const { changes } = await generateChangelog({ mode: 'init', mergePrerelease: true })
```

Comment:
Don't you think the mergePrerelease should be truthy (default) ?
In the CLI maybe ?

Reply:
Agree, it should be opt-in parameter